src/routes/user/_user.tsx

```diff
@@ -70,7 +74,12 @@ function LayoutComponent() {
       <div className="flex flex-col sm:gap-4 sm:py-4 sm:pl-14">
         <header className="sticky top-0 flex h-14 items-center gap-4 border-b bg-background px-4 sm:static sm:h-auto sm:border-0 sm:bg-transparent sm:px-6">
           <Sheet>
-            <h1>SPV Wallet Admin</h1>
+            <h1>SPV Wallet User</h1>
+            {userInfo && (
+              <p className="text-sm text-gray-600">
+                {userInfo.paymail ? `Paymail: ${userInfo.paymail}` : `User ID: ${userInfo.userId}`}
+              </p>
+            )}
```

Comment:
- You don't setting the paymail anywhere so maybe let's skip this. 
- Since you added `userId` field to `spvWalletClient` context you can skip the `useEffect` and just print the value from the context, like this:

 {spvWalletClient?.userId != null && `User ID: ${spvWalletClient.userId}`}

```suggestion
            <h1>SPV Wallet User</h1>
            {spvWalletClient?.userId != null && (
              <p className="text-sm text-gray-600">
                {`User ID: ${spvWalletClient.userId}`}
              </p>
            )}
```

Reply:
done